src/jquery.fancytree.dnd.js

```diff
@@ -312,22 +316,22 @@ $.ui.fancytree.registerExtension({
 				.prependTo($(this.$div).parent());
 //                .prependTo("body");
 
-			if( glyph ) {
+			if( glyphOpt ) {
 				// instData.$dropMarker.addClass(glyph.map.dragHelper);
 				instData.$dropMarker
-					.addClass(glyph.map.dropMarker);
+					.addClass(glyphOpt.map.dropMarker);
 			}
 		}
 		if( hitMode === "after" || hitMode === "before" || hitMode === "over" ){
-			markerOffsetX = -24;
+			markerOffsetX = dndOpt.dropMarkerOffset || 0;
 			switch(hitMode){
 			case "before":
 				markerAt = "top";
-				markerOffsetX -= 16;
+				markerOffsetX += (dndOpt.dropMarkerOffsetBefore || 0);
 				break;
 			case "after":
 				markerAt = "bottom";
-				markerOffsetX -= 16;
+				markerOffsetX += (dndOpt.dropMarkerOffsetAfter || 0);
```

Comment:
I thought about positive values, but if before/after offsets are evaluating as a sum of two options (dropMarkerOffset+dropMarkerOffsetAfter) then it'd be unclear for me what to specify as dropMarkerOffsetAfter: negative value (-16) or positive (+16).
I think I'd make sense only if we'll have two separated options (offsetOver/offsetInsert) which won't be summed.

>This offset will always be the same for 'after' and 'before' so I suggest to combine it into one single option.

Ok

> Also we could use two positive defaults:
dropMarkerOffsetX: 8
dropMarkerOffsetIndentX: 16

I'm sorry, I don't understand, why 8/16? They are -24 and -16 currently, aren't them?